shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -142,14 +144,14 @@ - (void)checkReadiness {
   }
 }
 
-- (NSString*)pathForFLXFromBundle:(NSBundle*)bundle {
-  NSString* flxName = [bundle objectForInfoDictionaryKey:@"FLTFlxName"];
-  if (flxName == nil) {
-    // Default to "app.flx"
-    flxName = @"app";
+- (NSString*)pathForFlutterAssetsFromBundle:(NSBundle*)bundle {
+  NSString* flutterAssetsName = [bundle objectForInfoDictionaryKey:@"FLTFlxName"];
```

Comment:
Is this still the right key, or should we use something like `FLTAssetsPath`?

(The key isn't defined in our template's Info.plist, so I don't know if anyone's actually using this.)

Reply:
I haven't been able to find any usages. I'll keep it with an updated key name so that it is still configurable in Info.plist.